libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -179,6 +182,9 @@ - (id) initWithConfig:(SFHybridViewConfig *) viewConfig useUIWebView:(BOOL) useU
     self = [super init];
     if (self) {
         self.useUIWebView = useUIWebView;
+        if (useUIWebView) {
```

Comment:
The current way the query should be straightforward (look for hybrid apps with `UI`). I'm not logging `WK` because that's the default and will become the only option soon, so we don't really care about that. We only want to know which apps are specifically using our new API to switch to `UIWebView`. Maybe we could use different letters to specify `UIWebView` instead of `UI`?